eng/Build.ps1

```diff
@@ -228,6 +233,7 @@ function BuildSolution([string] $solutionName) {
         /p:QuietRestoreBinaryLog=$binaryLog `
         /p:TestTargetFrameworks=$testTargetFrameworks `
         /p:DotNetBuildFromSource=$sourceBuild `
+        /p:CompressAllMetadata=$CompressAllMetadata `
```

Comment:
Seems an inconsistent name?   One or the other :) 

* "compress metadata"  or
* "compress all metadata" or
* "compress fsharp metadata" or 
* "compress resources"

Reply:
Oh I understand now, it's sort of an override setting.